exchange/exchange.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/prebid/prebid-server/prebid_cache_client"
 )
 
+type DebugContextKey string
```

Comment:
It needs to be declared in this fashion in order to store it in the context.

Reply:
In order to be able to store in the `Context`, string keys need to be their own types.

Reply:
According to https://golang.org/pkg/context/
"_The provided key must be comparable and should not be of type string or any other built-in type to avoid collisions between packages using context._ "